monocle/db/queries.py

```diff
@@ -433,6 +433,24 @@ def change_merged_avg_commits(es, index, repository_fullname, params):
     return data['aggregations']['agg1']['value']
 
 
+def changes_with_tests_ratio(es, index, repository_fullname, params):
+    params = deepcopy(params)
+    params['etype'] = ("Change",)
+    all = count_events(es, index, repository_fullname, params)
+    if all == 0:
+        return 0
+    params['files'] = Detector.tests_regexp
```

Comment:
Why don't you use the tests_included params for this ?

Reply:
done